core/assets/bundles/bundle.properties

```diff
@@ -1244,8 +1247,8 @@ block.plastanium-wall.description = A special type of wall that absorbs electric
 block.plastanium-wall-large.description = A special type of wall that absorbs electric arcs and blocks automatic power node connections.\nSpans multiple tiles.
 block.thorium-wall.description = A strong defensive block.\nDecent protection from enemies.
 block.thorium-wall-large.description = A strong defensive block.\nDecent protection from enemies.\nSpans multiple tiles.
-block.phase-wall.description = A wall coated with special phase-based reflective compound. Deflects most bullets upon impact.
-block.phase-wall-large.description = A wall coated with special phase-based reflective compound. Deflects most bullets upon impact.\nSpans multiple tiles.
+block.phase-wall.description = A wall coated with special phase-based reflective compound. Deflection chance is equal to the ratio of bullet damage to base deflection chance.
```

Comment:
Don't change the descriptions. This is wordy and will be rewritten anyway.

Reply:
Alright. I reverted the description changes.